lib/periodic_jobs.rb

```diff
@@ -92,6 +92,9 @@
   mgr.register('* 7 * * *', 'SignIn::DeleteExpiredSessionsJob')
   # Delete expired sessions
 
+  mgr.register('* 4 * * *', 'SignIn::CertificateCheckerJob')
```

Comment:
If the goal is once a day you need to make it '0 4 * * *'

Reply:
Also, this made me look at that job right above this, our `DeleteExpiredSessionsJob`, runs every minute. Dunno how that happened. It's not related to this PR but can you change that periodic job as well to be `0 7 * * *`